lang/python/python3-host.mk

```diff
@@ -22,23 +21,7 @@ HOST_PYTHON3_BIN:=$(HOST_PYTHON3_DIR)/bin/python$(PYTHON3_VERSION)
 
 HOST_PYTHON3PATH:=$(HOST_PYTHON3_LIB_DIR):$(HOST_PYTHON3_PKG_DIR)
 
-define HostPython3
```

Comment:
i think this was among the first HostPython functions, but then things got copied/duplicated a few times;
fine to remove it;